packages/decorators/README.md

```diff
@@ -1,5 +1,10 @@
-<div style="text-align:center;"><h1>@sapphire/decorators</h1>
-<h3>Useful TypeScript decorators for Sapphire Framework Discord bots</h3></div>
+<div align="center">
+
+![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire.png)
+
+# @sapphire/decorators
+
+Useful TypeScript decorators for Sapphire Framework Discord bots</h4>
```

Comment:
```suggestion
Useful TypeScript decorators for Sapphire Framework bots.</h4>
```
Anybody looking at this would likely know that Sapphire is for Discord bots